spec/lib/appsignal/integrations/mongo_ruby_driver_spec.rb

```diff
@@ -81,7 +81,7 @@
         transaction.should receive(:finish_event).with(
           'query.mongodb',
           'find | test | SUCCEEDED',
-          "{\"foo\":\"?\"}",
+          instance_of(Appsignal::Extension::Data),
```

Comment:
I see we have specs that test the generator, but wouldn't be good to see if the data we expect in the `Data` object to be what we expect?

Can't really create a solution for it since I can't install the extension. See also the travis build